xml/System.Text.Json.Serialization/JsonConverterAttribute.xml

```diff
@@ -102,8 +114,8 @@
       </Parameters>
       <Docs>
         <param name="typeToConvert">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>If overridden and <see cref="P:System.Text.Json.Serialization.JsonConverterAttribute.ConverterType" /> is <see langword="null" />, allows a custom attribute to create the converter in order to pass additional state.</summary>
```

Comment:
@ahsonkhan when you say "custom" you mean a class that implements JsonConverterAttribute?
```suggestion
        <summary>When overridden in a derived class and <see cref="P:System.Text.Json.Serialization.JsonConverterAttribute.ConverterType" /> is <see langword="null" />, allows the derived class to create a <see cref="T:System.Text.Json.Serialization.JsonConverter"/> in order to pass additional state.</summary>
```